internal/bus/topics.go

```diff
@@ -19,6 +19,6 @@ const (
 	WriteConfigSuccessfulTopic = "write-config-successful"
 	ConfigApplySuccessfulTopic = "config-apply-successful"
 	ConfigApplyFailedTopic     = "config-apply-failed"
-	RollbackCompleteTopic      = "rollback-complete"
+	ConfigApplyCompleteTopic   = "config-apply-complete"
```

Comment:
wondering if adding `rollback` in the name make it better?

Reply:
The naming was changed as it’s no longer just used in Rollback, it is one of two ways to clear the cache, and restart the watchers which are paused during config apply. There were scenarios where config apply is done but it wasn’t a rollback or successful scenario. Example would be config apply failing before every writing files like an error when determining the file actions. In this case no rollback is required, config apply wasn’t successful but the config apply is done.

Reply:
I see, thanks for the example.